File: Python Word examples/tables/delete_table_row.py
import spirecloudword
from spirecloudword.api.tables_api import TablesApi
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.tables_api.TablesApi(configuration)

filename = "Template.docx"
table_path = "sections/0/tables/0"
index = 0
inputFolder = "input"
outputFilePath = "output/Tables/delete_table_row.docx"
result = api.delete_table_row(filename, table_path, index, folder=inputFolder, dest_file_path=outputFilePath)

File: JavaScript Word Examples/TablesApiDemo/Rows/deleteTableRow.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TablesApi(apiClient); 

    var filename = "Table.docx";
    var tablePath = "Section/0/Body/0/Table/0";
    var desFilePath = "output/Tables/DeleteTableRow_1.docx";
    var index = 0;
    var opts = {folder: 'Tables'};
    instance.deleteTableRow(filename, tablePath, index, desFilePath, opts,
    function (error, data, response) {
         if (error) throw error;
    });
})();